adam/learner/attributes.py

```diff
@@ -297,3 +304,192 @@ def _update_hypothesis(
             ),
             trim_after_match=pattern_remove_incomplete_region_or_spatial_path,
         )
+
+
+@attrs
+class PursuitAttributeLearnerNew(  # should it be called new? there is no old one as far as I know
```

Comment:
I'd suggest leaving it called new given the rest of the current learners we are using are called 'new'. Before we release the software if we have time we should probably clean this up.